src/BuildScriptGeneratorCli/Commands/BuildCommandBase.cs

```diff
@@ -33,6 +33,12 @@ internal abstract class BuildCommandBase : CommandBase
             Description = "Comma-separated list of operating system packages that will be installed (using apt-get) before building the application.")]
         public string OsRequirements { get; set; }
 
+        [Option(
+            OptionTemplates.AppType,
+            CommandOptionType.SingleValue,
+            Description = "Application type in a multi-app repo, for example a repo that has functions api and blazor-wasm client app.")]
```

Comment:
`Type of application that the source directory has, for example: 'functions' or 'static-sites' etc.`